### ISSUE TITLE
Bug geneypes not visible

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#1859]
@CuzickA  was experiencing a bug

where the geneotypes were  not visible Only the headings were there)

This is what I see
![i see](https://user-images.githubusercontent.com/7359272/57530037-4122cb80-732e-11e9-95ad-0b6d78ba21c8.jpg)

For the same session only the headings were visible for Alayne

**Comment by CuzickA** [comment]
When I follow the curation link http://canto.phi-base.org/curs/bd02fdb6831712ca
This is what I see
<img width="942" alt="cant see genotypes" src="https://user-images.githubusercontent.com/11753634/57530583-7d0a6080-732f-11e9-94a8-327c876f4886.PNG">

To get around this I can go to the All publications in Admin, select the pubmed and follow link to the curation session. From here I can see the genotypes.

I saw this error on both my laptop and desktop.

I have just followed the link again on both machines and the genotypes are visible. may have been a temporary glitch.
